yarn-project/p2p/src/mocks/index.ts

```diff
@@ -26,16 +26,15 @@ import { type BootnodeConfig, type P2PConfig } from '../config.js';
 import { type MemPools } from '../mem_pools/interface.js';
 import { DiscV5Service } from '../services/discv5/discV5_service.js';
 import { LibP2PService } from '../services/libp2p/libp2p_service.js';
-import { type PeerManager } from '../services/peer_manager.js';
```

Comment:
A lot of diffs as i moved peer-scoring and peer-manager into the peer-manager service folder